lib/extensions.js

```diff
@@ -28,10 +30,26 @@ function getRuntimeInfo() {
  */
 
 function getBinaryIdentifiableName() {
+  var v8SemVersion = process.versions.v8;
```

Comment:
Shouldn't it be `.slice(0, 3)` to extract semver?

In some releases (for instance node v0.12.0), there are three portions (3.28.73). While in others (all iojs releases till date) have four portions.

I thought it would be good to save a join when v8 version already have three portions.